src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -88,6 +88,19 @@
     private final Set<Integer> javadocTokens = new HashSet<>();
 
     /**
+     * This property determines if a check should log a violation upon encountering javadoc with
+     * non-tight html. The default return value for this method is set to false since checks
+     * generally tend to be fine with non tight html. It can be set through config file if a check
+     * is to log violation upon encountering non-tight HTML in javadoc.
+     *
+     * @see ParseStatus#firstNonTightHtmlTag
+     * @see ParseStatus#isNonTight()
+     * @see <a href="http://checkstyle.sourceforge.net/writingjavadocchecks.html#Tight-HTML_rules">
+     *     Tight HTML rules</a>
+     */
+    private boolean violateExecutionOnNonTightHtml;
```

Comment:
> Current XDoc verification won't allow this as we ignore all properties inside the base classes by default.

lets make a hack in that test and do workaround/skip extra items from this class. We can do proper  fix in sepparte issue.